django_prices_vatlayer/utils.py

```diff
@@ -0,0 +1,45 @@
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils import six
+
+from .models import Vat
+
+
+try:
+    ACCESS_KEY = settings.VATLAYER_ACCESS_KEY
+except AttributeError:
+    raise ImproperlyConfigured('VATLAYER_ACCESS_KEY is required')
+
+RATINGS_URL = 'http://apilayer.net/api/rate_list'
+
+
+def validate_data(json_data):
+    if not json_data['success']:
+        info = json_data['error']['info']
+        raise ImproperlyConfigured(info)
```

Comment:
We will get wrong response only when `ACCESS_KEY` is not proper so I think that it is more `ImproperlyConfigured` than `ValueError`